fmt/duration_test.ts

```diff
@@ -85,7 +85,7 @@ Deno.test({
   fn() {
     assertThrows(
       () => {
-        format(16.342, { style: undefined });
+        format(16.342, { style: undefined } as object);
```

Comment:
```suggestion
        format(16.342, {});
```

Reply:
This test is incorrect so I will fix it in another PR.

Reply:
#5893